### ISSUE TITLE
Update to KEDA 2.0

### ISSUE THREAD

**Issue by jeffhollan** [NileshGule/pd-tech-fest-2019#34]
We'll be updating the [KEDA Samples README](https://github.com/kedacore/samples) to call out which samples only work with KEDA 1.0.  It looks like you'd need changes to make this 2.0 compatible (in deployment), so creating issue here to call out.  Here's a link to the migration guide for 2.0 https://keda.sh/docs/2.0/migration/

Thanks regardless for this sample!

**Comment by NileshGule** [comment]
@jeffhollan I have made the changes as suggested in the migration from KEDA 1.0 to 2.0 document. When I use Helm charts to install KEDA on Kubernetes v1.19.0 cluster, it is failing. The issue is same as 
https://github.com/kedacore/keda/issues/1310

I tried installing the KEDA 2.0 RC version using manifest files but the HPA for my deployment is not getting created. 
Is there any working example which works end to end for me to refer?
